Task 3/Linkedlist.kt

```diff
@@ -0,0 +1,145 @@
+class Node<T>(value: T){
```

Comment:
Please mention which kind of Linked List and create a folder and insert the program file inside it.

Reply:
Folder structure created and Readme file of Linkedlist provided.Approve the PR upon review